packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -754,7 +805,8 @@ class _PageViewState extends State<PageView> {
         physics: physics,
         viewportBuilder: (BuildContext context, ViewportOffset position) {
           return Viewport(
-            cacheExtent: 0.0,
+            cacheExtent: widget.allowImplicitScrolling ? 1.0 : 0.0,
```

Comment:
Maybe leave a TODO here to make the cacheExtend/cacheExtendStyle configurable on the pageView once we have implicitScrolling set up correctly on iOS.

Reply:
Done